addons/hr/models/hr_employee_public.py

```diff
@@ -30,6 +30,8 @@ class HrEmployeePublic(models.Model):
     resource_id = fields.Many2one(readonly=True)
     tz = fields.Selection(readonly=True)
     color = fields.Integer(readonly=True)
+    partner_fields = fields.Json(readonly=True)
+    primary_email_field = fields.Char(readonly=True)
```

Comment:
Note: Those fields are necessary to avoid issues with [`_check_private_fields`](https://github.com/odoo-dev/odoo/blob/master-recipient-fields-did/addons/hr/models/hr_employee.py#L424)